includes/definitions/discovery/fujitsuiRMC.yaml

```diff
@@ -0,0 +1,106 @@
+mib: FSC-SERVERCONTROL2-MIB
+modules:
+    os:
+        hardware: FSC-SERVERCONTROL2-MIB::sc2UnitModelName.1 # if MIB is not named here, value is not displayed. 
+        version: FSC-SERVERCONTROL2-MIB::sc2spModelName.1.1 # if MIB is not named here, value is not displayed. 
+        serial: FSC-SERVERCONTROL2-MIB::sc2UnitSerialNumber.1 # if MIB is not named here, value is not displayed. 
+    sensors:
+        state:
+            data:
+                -
+                    oid: sc2AgentStatus
+                    value: sc2AgentStatus
+                    num_oid: '.1.3.6.1.4.1.231.2.10.2.2.10.8.1.{{ $index }}'
+                    descr: 'Global System Status'
+                    #index: 'statusSummary.{{ $index }}'
```

Comment:
How come you've commented the index key value pair out?

Reply:
This was a leftover from my tests for a correct yaml and correct test data. Removed this.